mex/common/wikidata/connector.py

```diff
@@ -77,6 +78,7 @@ def get_wikidata_item_details_by_id(self, item_id: str) -> dict[str, str]:
             ),
             "formatversion": "2",
         }
+        headers = {"User-Agent": "rki/mex", "Api-User-Agent": "rki/mex"}
```

Comment:
I vote for DRY here. Would be better to put this into the mex.common.settings

Reply:
will something like this work? 5d4e668b8103bed123edb109824b19a2899949a9

entire dict can vary from service to service, so it wont be reuseable everywhere if we move it to settings.